packages/common/src/ui/components/inputs/TextInput/ReadOnlyInput.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+export interface ReadOnlyInputProps {
+  number?: boolean;
+  style?: React.CSSProperties;
+  width?: string;
+}
+
+export const ReadOnlyInput: React.FC<ReadOnlyInputProps> = React.forwardRef<
```

Comment:
possibly a daft idea. I wanted to add text which is controlled by the form, and which didn't look like an input.

Reply:
> Right now looks just like a disabled input

Only used in the batch table currently - to show the unit quantity here:

<img width="417" alt="Screen Shot 2021-11-01 at 12 33 52 PM" src="https://user-images.githubusercontent.com/9192912/139604942-54eaa801-ef63-4909-baab-262f8e1ae276.png">

> I wondered about a more informational approach

yeah - that's what I was thinking. it troubled me having the disabled inputs. I've kept them so far because they show the distinction between label and value pretty well and also make the form a little less boring.